core/src/scene/drawRule.h

```diff
@@ -37,11 +38,10 @@ class StyleContext;
 struct DrawRuleData {
 
     std::vector<StyleParam> parameters;
-    std::string name;
-    int id;
+    std::string styleName;
+    int styleId;
```

Comment:
Oh wow the docs are suuuper misleading here. I'll bring up this nomenclature with the JS and docs folks so we can clear it up. I don't think we should adapt our code to mimic confusing documentation, we should rather clarify the documentation itself.